### DISCUSSION TITLE
Moonraker error

### DISCUSSION THREAD

**Discussion by novaprime73** [community/community#86712] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello Sir 
I am getting moonraker warning and was wondering if you can assist in solving them 

Here is the warning 
Moonraker warnings found.
Server configuration error: Duplicate section [update_manager] in file /usr/data/printer_data/config/moonraker.conf
Loaded server from most recent working configuration: '/usr/data/printer_data/config/.moonraker.conf.bkp'
Please fix the issue in moonraker.conf and restart the server.

Here is the lines from my config file 

[update_manager]
enable_auto_refresh: True
refresh_interval: 24
enable_system_updates: False

 Remove '#' from the lines below if you use KAMP
[update_manager]
type: git_repo
channel: dev
path: /usr/data/KAMP-for-K1-Series
origin: https://github.com/Guilouz/KAMP-for-K1-Series.git
primary_branch: main
is_system_service: False

an assistance is appreciated thank you

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
